pkg/build/node/node.go

```diff
@@ -263,6 +294,92 @@ func (c *BuildContext) buildImage(dir string) error {
 	return nil
 }
 
+// must be run after kubernetes has been installed on the node
+func (c *BuildContext) prePullImages(dir, containerID string) error {
+	// first get the images we actually built
+	builtImages, err := c.getBuiltImages()
+	if err != nil {
+		log.Errorf("Image build Failed! %v", err)
+		return err
+	}
+
+	// helpers to run things in the build container
+	execInBuild := func(command ...string) error {
```

Comment:
execing in general needs a cleanup in a follow-up, I have some ideas that will make the exec package much nicer.